roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -1,14 +1,13 @@
 {
-    "context_is_admin": "role:admin or role:cloud_admin",
-    "cloud_admin_privilege": "role:admin or role:cloud_admin",
-    "is_owner": "project_id:%(tenant_id)s or project_id:%(project_id)s",
-    "admin_or_owner": "rule:cloud_admin_privilege or rule:is_owner or rule:project_admin_privilege",
+    "context_is_admin": "role:admin",
+    "owner": "tenant_id:%(tenant_id)s or project_id:%(tenant_id)s or project_id:%(project_id)s",
```

Comment:
The `project_id:%(tenant_id)s` and `project_id:%(project_id)s` checks were necessary in Kilo for Horizon. Added here for immediate compatibility, but this will need to be tested later to see if we can remove them.